core/src/main/java/jenkins/security/LastGrantedAuthoritiesProperty.java

```diff
@@ -148,10 +148,6 @@ protected void loggedOut(@Nonnull String username) {
 
     @Extension
     public static final class DescriptorImpl extends UserPropertyDescriptor {
-        public String getDisplayName() {
-            return null;    // not visible
```

Comment:
IIRC in some contexts (properties?), `null` means _do not display_. This seems to be the case. I doubt it is documented anywhere, though there are plugins in the wild that rely on that.